registry/client/repository.go

```diff
@@ -377,11 +377,7 @@ func (bs *blobs) Stat(ctx context.Context, dgst digest.Digest) (distribution.Des
 }
 
 func (bs *blobs) Get(ctx context.Context, dgst digest.Digest) ([]byte, error) {
-	desc, err := bs.Stat(ctx, dgst)
```

Comment:
Interesting. I'm trying to see where this happens in the code, and I can't find it. `blobStatter.Stat` in blobstore.go seems to return the digest argument without any modification in the descriptor it returns.